src/main/scala/edu/berkeley/ce/rockslicing/Joint.scala

```diff
@@ -204,12 +204,15 @@ object Joint {
   *        This avoids recalculation of a known dip angle.
   * @param dipDirectionParam An optional parameter that can be used to specify the dip direction for
   *        the joint. This avoids recalculation of a known dip direction.
+  * @param artificialJoint Parameter that identifies joint as being artificial joint introduced as part
+  *                        of load balancing
   */
 case class Joint(normalVec: (Double, Double, Double), localOrigin: (Double, Double, Double),
                  center: (Double, Double, Double), phi: Double, cohesion: Double,
-                 shape: Seq[((Double, Double, Double),Double)], dipAngleParam: Option[Double]=None,
+                 shape: List[((Double, Double, Double),Double)], dipAngleParam: Option[Double]=None,
```

Comment:
Why is this changed to `List`? That was what we had originally, but then I went back and changed things to `Seq` a while ago because it appears to be the recommended practice that we use the `Seq` interface and let Scala pick a concrete implementation.

Is there something we need to do that requires a `List`? I wonder if we couldn't a way to achieve the same thing with `Seq` after doing some research.